app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\User;
+use Auth;
+
+class UserController extends Controller
+{
+    public function edit()
+    {
+        if(Auth::user()) {
```

Comment:
This should be unnecessary because this route should be protected by auth middleware.